publisher/internal/analytics/ethereum/analytics.go

```diff
@@ -27,20 +22,26 @@ var eventsToAvoid = []string{
 	"TransformedERC20(address,address,address,uint256,uint256)",
 }
 
+var signaturesToAvoid map[string]struct{}
+
 const (
 	subSubject            = "syntropy.ethereum.log-event"
 	uniswapPositionsOwner = "0xC36442b4a4522E871399CD717aBDD847Ab11FE88"
+	mintSig               = "0x7a53080b" //Mint(address,address,int24,int24,uint128,uint256,uint256)
```

Comment:
Maybe it would be better UX for future developers if these constants were calculated in the `init` function from human-readable strings using `convertToEventSignature`?